fieldservice_sale/views/res_config_settings.xml

```diff
@@ -1,11 +1,13 @@
 <odoo>
 
-    <record model="res.config.settings" id="res_config_settings_fsm_template">
+    <!-- ToDo -->
+
+    <!-- <record model="res.config.settings" id="res_config_settings_fsm_template">
         <field name="group_fsm_template" eval="1" />
     </record>
 
     <function model="res.config.settings" name="execute">
             <value eval="[ref('fieldservice_sale.res_config_settings_fsm_template')]" />
```

Comment:
You can try using set_values instead.
Note: both `execute` and `set_values` don't have any arguments...
```suggestion
    <function model="res.config.settings" name="set values">
```